frontend/src/pages/GranteeRecord/index.css

```diff
@@ -1,3 +1,9 @@
+.ttahub-grantee-record--heading {
+    font-size: 40px;
+    font-family: "Merriweather", serif;
+    font-weight: 700;
+}
+
```

Comment:
Trying to prevent the scope of this from creeping anymore, but I find myself reusing this code a lot from the h1.landing, expect removing the no wrap behavior. (I can't figure out why we do that) 

Just a note to self to abstract these three into a ttahub-page-heading or some such